fud2/fud-core/Cargo.toml

```diff
@@ -24,4 +24,5 @@ once_cell = "1.19.0"
 ariadne = "0.4.1"
 itertools.workspace = true
 rand = "0.8.5"
+rand_chacha = "0.3.1"
```

Comment:
Possibly silly question, but any particular need for a specific PRNG? Surely we don't need cryptographic PRNGs for this?

Reply:
The specific prng doesn't matter, but I want the test to always reproduce (i.e. an update to `rand`s prng which changes their implementation shouldn't break the tests).